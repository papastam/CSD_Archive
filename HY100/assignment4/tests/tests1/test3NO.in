  3
  2 7 1
  9 5 1
  4 3 8
