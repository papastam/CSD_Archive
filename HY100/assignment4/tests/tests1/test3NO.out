ROW 1 NO
ROW 2 YES
ROW 3 YES
COLUMN 1 YES
COLUMN 2 YES
COLUMN 3 NO
DIAG1 YES
DIAG2 NO
UNIQUE NO
MAGIC NO
